include/ddsrouter/configuration/DDSRouterConfiguration.hpp

```diff
@@ -75,8 +75,10 @@ class DDSRouterConfiguration : public BaseConfiguration
      * invalid \c ParticipantType ) will be added to the list.
      *
      * @return List of \c ParticipantConfigurations
+     *
+     * @throw \c ConfigurationException in case a Participant is not well configured (e.g. No type)
```

Comment:
Now this should also be added to DDSRouter::init_participants_

Reply:
:+1: